src/services/time.js

```diff
@@ -0,0 +1,10 @@
+import moment from 'moment-timezone';
+
+export const getUKTimestamp = () => {
+  return moment().tz('Europe/London').format('YYYY-MM-DDThh:mm:ssZ');
+};
```

Comment:
Optional suggestion!

```suggestion
export const getUKTimestamp = () => moment().tz('Europe/London').format('YYYY-MM-DDThh:mm:ssZ');
```